README.rst

```diff
@@ -28,18 +28,18 @@ If you have deployed client-side STF and want to test it locally, you can run th
 
     ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml
 
-OR
+OR::
 
     ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml --tags "<one of the tags from above>"
 
-To get a summarised output of the tests at the end, whitelist the provided logging callback.::
+To get a summarised output of the tests at the end, use the provided logging callback::
 
     ANSIBLE_CALLBACK_WHITELIST=custom_logger ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml
 
-The logging callback will summarise the tests run on each node and whether they passes or failed.
-The callback will only report on the status of tasks that have a name beinging with "[Test]".
+The logging callback will summarise the tests run on each node and whether they passed or failed.
+The callback will only report on the status of tasks that have a name beginning with ``[Test]``.
 
-Note::
+.. note::
```

Comment:
This still doesn't look like it renders how you want

Reply:
I think we need some sphinx build of style sheets to do this properly, but this will have it indicated as a note. It's good enough for the moment.

Reply:
Also looking at the examples I have found, the note should start on the same line as the admonition is defined:

.. note:: If you haven't deployed using infrared, you can still run the tests if you create your own inventory file, containing one group of hosts called ``overcloud_nodes``.
   Alternatively, you can create your own playbook that imports the tasks in ``tasks/*.yml``

Reply:
And of course I found _another_ example. All the indenting looks like it should align to 3 spaces in (alignment on the newline starting at the `n` of note.

Anyways, more just a curiosity, go ahead and merge when you see fit :)